### ISSUE TITLE
undefined is not an object (evaluating '_this.menu.current.show')

### ISSUE THREAD

**Issue by kas2000** [mxck/react-native-material-menu#18]
When I try to click on show menu button the error occurs: undefined is not an object (evaluating '_this.menu.current.show')

Code:

```
`import Menu, { MenuItem, MenuDivider } from 'react-native-material-menu';
export default class Apartments_Advert extends React.Component{
  menu = React.createRef();

  hideMenu = () => {
    this.menu.current.hide();
  };

  showMenu = () => {
    this.menu.current.show();
  };
 
 render(){
    return(
        <Menu
              ref={this.menu}
              button={<Text onPress={this.showMenu}>Show menu</Text>}
            >
              <MenuItem onPress={this.hideMenu}>Menu item 1</MenuItem>
              <MenuItem onPress={this.hideMenu}>Menu item 2</MenuItem>
              <MenuItem onPress={this.hideMenu} disabled>
                Menu item 3
              </MenuItem>
              <MenuDivider />
              <MenuItem onPress={this.hideMenu}>Menu item 4</MenuItem>
         </Menu>
)
 }

}
`
```

**Comment by mxck** [comment]
@kas2000 can you post your `react-native info`?

**Comment by phe-move-elevator-de** [comment]
Hy, i have the same issue. Here is my info:

Environment:
  OS: macOS High Sierra 10.13.4
  Node: 7.10.0
  Yarn: Not Found
  npm: 4.2.0
  Watchman: 4.9.0
  Xcode: Xcode 9.3 Build version 9E145
  Android Studio: 3.0 AI-171.4443003

Packages: (wanted => installed)
  react: 16.3.0-alpha.1 => 16.3.0-alpha.1
  react-native: 0.54.1 => 0.54.1

**Comment by mxck** [comment]
@phe-move-elevator-de @kas2000 ok. I check this on Monday :)

**Comment by mxck** [comment]
Fixed in 7a7d8132ac0b5c6f7892801b1cb92a51cc022883
